src/main/java/frc/robot/controls/Operator.java

```diff
@@ -89,9 +95,10 @@ public Operator(Robot robot)
         climberUnwind.whenHeld(new ClimberUnwind(robot));
         climberExtend.whenHeld(new ClimberExtend(robot));
         climberRetract.whenHeld(new ClimberRetract(robot));
-    }
-
-	// METHODS //
 
+        shooterSpin.whenHeld(new ShooterSetSpeed(robot));
+        shooterSpin.whenReleased(new ShooterSpin(robot));
```

Comment:
How do these controls work?  When you hold down the button, shouldn't the shooter spin?  When released, it can stop, but you already set the default command to ShooterStop so you don't even need to code that in.